letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -459,7 +440,12 @@ def _create_vhost(self, path):
         filename = get_file_path(path)
         is_enabled = self.is_site_enabled(filename)
 
-        vhost = obj.VirtualHost(filename, path, addrs, is_ssl, is_enabled)
+        macro = False
+        if "/macro/" in path.lower():
```

Comment:
Perhaps `macro = ("/macro/" in path.lower())`?

Or `macro = True if ("/macro" in path.lower()) else False`?